OWNERS

```diff
@@ -5,6 +5,7 @@ approvers:
 - technical-oversight-committee
 - serving-wg-leads
 - serving-writers
+- autoscaler-keda-approvers
 
 reviewers:
 - serving-reviewers
```

Comment:
I could replace this with `autoscaler-keda-approvers` to avoid noise but also it might be useful if others want to review stuff.